src/components/balances/TokenListSelect/index.tsx

```diff
@@ -58,8 +54,8 @@ const TokenListSelect = () => {
           label="Tokenlist"
           onChange={handleSelectTokenList}
           renderValue={(value) => TokenListLabel[value]}
-          onOpen={() => handleTrack(TokenListActions.OPEN)}
-          onClose={() => handleTrack(TokenListActions.CLOSE)}
+          onOpen={() => handleTrack(ASSETS_EVENTS.OPEN_TOKEN_LIST_MENU)}
+          onClose={() => handleTrack(ASSETS_EVENTS.CLOSE_TOKEN_LIST_MENU)}
```

Comment:
On a second thought, do we really want to track when it's closed? What insight can you get from this?

Reply:
Valid point. I saw the `CURRENCY_MENU` was tracking both events so I applied the same criteria but it brings little value indeed.